### DISCUSSION TITLE
[ 求助 ] JS脚本逆向

### DISCUSSION THREAD

**Discussion by DeepslateQAQ** [share121/inter-knot#41] [其他]
![image](https://github.com/user-attachments/assets/6ba6dbdb-2d43-4774-ae64-bcea0baa06ef)

各位绳匠们，手头有一份JavaScript脚本，里面的变量名全都看不懂（绝对不是邦布语！！），刚刚改完一部分，有绳匠会弄这个么

**Comment by MouxiuSama** [comment]
这位绳匠真是厉害，本人都看不懂呢，不过多等等说不定有大佬

**Comment by yuugaochyan** [comment]
看着是ob混淆，找一下大数组、移位函数和解密函数，然后全部自动替换一下。

**Comment by DeepslateQAQ** [comment]
> 看着是ob混淆，找一下大数组、移位函数和解密函数，然后全部自动替换一下。

大数组啥的解密完了，剩下一堆`_0xXXXXXX`的名称，分析不出来
